rke/k8s_version_info.go

```diff
@@ -146,9 +146,9 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
-		"v1.16.10-rancher2-3": {
-			MinRancherVersion: "2.4.6-rc0",
-			MinRKEVersion:     "1.1.4-rc0",
+		"v1.16.13-rancher1-1": {
+			MinRancherVersion: "2.3.8-rc0",
```

Comment:
Should this be 2.4.4, since we have a bug in KDM that prevents dynamic KDM releases which was fixed in 2.4.4?

Reply:
I put this up so rancher setups upgrading from 2.3.8+ (with 1.16.13-rancher1-1) to 2.4.0+ don't face any issue. cc @deniseschannon

Reply:
For upgraded setups, the CRDs would already be loaded and 2.4.0-2.4.3 should continue functioning as is (unless they get deleted for some reason :)). Since we didn't handle this before, I'm going to keep it consistent and put min version of 2.4.4.

Reply:
Putting 1.16.x and 1.17.x to 2.4.4 -- barring any issues, this would allow users to get the latest patch versions without upgrading to 2.4.5.